api/routes.py

```diff
@@ -275,6 +276,12 @@ def decorator(f):
 def index():
     return app.manager.index_controller()
 
+
+@app.route('/', methods=['POST'])
+def return_post():
+    # Handle 405 error for POST method to '/'
+    return Response('Method not allowed', 405)
+
```

Comment:
Is there a need for this explicit route? Or could we just allow `POST` on root and allow the handler you put in the `__call__` method to handle it?

From an abstract perspective this is great, but it clashes a bit with the pattern established in this module.